### DISCUSSION TITLE
Not an issue, just wanted to say I loved the idea of your mod/datapack

### DISCUSSION THREAD

**Discussion by YumatanGames** [jacobsjo/lavaflow#2] [General]
And I'd also like to ask for a 1.21.1 version, if possible. I play Minecolonies, mostly, and would love to have a more realistic and interesting Nether for a Nether colony. Unfortunately, as of now at least, Minecolonies is only available for 1.21.1. Anyhow, is your mod naturally compatible with other mods that changes the Nether, like Biomes O' Plenty, Amplified Nether, Incendium and others? 

Thanks!

**Comment by jacobsjo** [comment]
Thank you! You reminded my I should open this discussion tab on discord :+1: 

As you might have seen, the ghost forest used pale moss and pale oak leaves, which makes it impossible to backport that biome to before 1.21.4. The remaining part would be possible, but will probably not happen anytime soon. I'm more targeting the next popular modpack version, whatever that might be.

Comparability is tricky. Since this datapack makes use of of a lot of overworld techniques, it makes comparability in the nether as tricky as it is in the overworld.

There will have to be special compatability datapacks that add other mods biomes to appropriate terrain in lavaflow. This isn't particularly difficult to do, but tricky the amount of possible combinations of mods. I'll figure out a solution eventually, but this will come with time.

Datapacks that add structures should be compatible, but might cause weird effects of structures floating. It would need some adjustment to their placement biomes and start-height to make it look good. Again, compatibility packs might come with time.

Datapacks that modify terrain (i.e. Amplified Nether) are just incompatible. There isn't really a point in using both.

**Reply by YumatanGames** [reply]
I see. I appreciate you being willing to add the compatibilities wherever possible. Maybe one day Minecolonies will be updated to 1.21.4+ so I might be able to play with your pack. For now, I'm just testing it and it really looks good. Does it increases Nether world height to match the overworld? Last time I played with your pack I didn't check that.

**Reply by jacobsjo** [reply]
It doesn't. It increased the nether roof by 16 blocks to y=144. The build height limit remains unchanged at y=256. That is a very easy thing to tweak yourself though.

**Reply by YumatanGames** [reply]
I'm just wondering because you could make your changes even more dramatic if you also increased the height (both bellow 0 and above 256) to match the overworld. :D
